pyrevitlib/pyrevit/compat.py

```diff
@@ -33,6 +33,24 @@
     from urllib.parse import urlparse
 
 
+def is_netcore():
```

Comment:
I think good idea to rewrite this code with using `isinstance`

```python

if isinstance(__revit__, Application):
  pass
else if isinstance(__revit__, UIApplication)
  pass
else if isinstance(__revit__, UIControlldedApplication)
  pass

// etc...
```

Reply:
@sanzoghenzo 
I don't like this option because the code is hard to read and it's not very clear why it's written that way, then it's worth adding comments about what types are theoretically checked and what types are possible there